api/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # This must be run in project root, to pick up dataset_config/.
-FROM python:2
+FROM python:3
```

Comment:
Please make a branch on this repo instead of your fork. Otherwise circleci won't run.

(Also, I'm not able to test some datasets because commit `6cee79dab14fc9707b5936345e35bd0b54578425 ` doesn't exist.)

Reply:
I will create a new PR from a branch in this repo.